app/components/tech-icons/InkIcon.tsx

```
@@ -2,32 +2,101 @@ import React from "react";
   return (
     <>
       <svg
         className={`hidden sm:block ${className}`}
         width={47}
         height={47}
-        viewBox="0 0 24 24"
+        viewBox={`-1 -1 ${width + 2} ${height + 2}`}
         fill="none"
         xmlns="http://www.w3.org/2000/svg"
         {...props}
       >
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)